src/App.tsx

```diff
@@ -1,7 +1,27 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export const App: React.FC = () => {
+  const [pressedKey, setPressedKey] = useState('');
+
+  useEffect(() => {
+    const handleDocumentKeyup = (event: KeyboardEvent) => {
+      setPressedKey(event.key);
+    };
+
+    document.addEventListener('keyup', handleDocumentKeyup);
+
+    return () => {
+      document.removeEventListener('keyup', handleDocumentKeyup);
+    };
+  }, []);
```

Comment:
It would have nearly zero effect, but technically you need to have your `setPressedKey` as a dependency to the effect hook